code/src/components/AmountOf.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const AmountOf = () => {
+  const todos = useSelector(state => state.todos.list.todos)
+  const leftTodo = todos.filter(todo => todo.complete === false)
+
+  //If everything is done, give pep!
+  if (leftTodo.length === 0) {
+    return (
+      <>
+      <p className="amountOf">Keep it up!</p>
+      <p className="amountOf pep">Nothing more to do right now.</p>
+      </>
+    )
+  }
+  
+  //If list is empty, don't show counter
+  //If nothing been done, encourage!
+  return (
+    <div>
+      <p className={todos.length === 0 ? "noAmount" : "amountOf"}>
```

Comment:
you can also check the length like this:
`<p className={!todos.length ? "noAmount" : "amountOf"}>`